meta-lmp-base/recipes-kernel/linux/linux-lmp_5.15.bb

```diff
@@ -15,3 +15,5 @@ KMETA = "kernel-meta"
 
 require linux-lmp.inc
 include recipes-kernel/linux/linux-lmp-machine-custom.inc
+
+DEFAULT_PREFERENCE = "-1"
```

Comment:
This snip is not required. we only need to set it when these recipe is the higher version and we prefer another one with a lower version.